src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -94,7 +94,7 @@
 
     <IncludeDotNetCli>true</IncludeDotNetCli>
     <DotNetCliPackageType>runtime</DotNetCliPackageType>
-    <DotNetCliVersion>$(BundledNETCoreAppPackageVersion)</DotNetCliVersion>
+    <DotNetCliVersion>5.0.0</DotNetCliVersion>
```

Comment:
Let's open an issue to remember to put this back to parse the global.json file or use `BundledNETCoreAppPackageVersion` so that we're on an up-to date SDK.

Reply:
Opened #44578